docs/SIL.rst

```diff
@@ -692,6 +692,18 @@ autorelease in the callee.
   type behaves like a non-generic type, as if the substitutions were
   actually applied to the underlying function type.
 
+- SIL functions may optionally mark a function parameter as
+  ``@sil_isolated``. An ``@sil_isolated`` parameter must be one of:
+
+  - An actor or any actor type.
+  - A generic type that conforms to Actor or AnyActor.
```

Comment:
AnyActor being a marker protocol, I am wondering if this means Actor or DistributedActor in practice?

Reply:
Not sure. What is important is that both have the semantics of Actors (namely they can create isolation).